File: Ruby/group.rb
class Group

	def initialize(result, operation)
		@result = result
		@operation = operation
		@cells = []
	end

	def cells
		@cells
	end

	def operation
		@operation
	end

	def result
		@result
	end

end

__END__
class Group:

	def __init__(self, result, operation = None):
		self.result = result
		self.operation = operation
		self.cells = []

	def add(self, cell):
		self.cells.append(cell)

File: Java/src/main/java/com/clovisleoncio/kenkensolver/Group.java
package com.clovisleoncio.kenkensolver;

import java.util.ArrayList;
import java.util.List;

import lombok.Data;

@Data
public class Group {

	private int result;
	private Operation operation;
	private List<Cell> cells = new ArrayList<Cell>();

	public Group(int result, Operation operation) {
		this.result = result;
		this.operation = operation;
	}

	public void add(Cell cell) {
		cells.add(cell);
	}

}
